pulpcore/app/migrations/0001_initial.py

```diff
@@ -436,7 +436,7 @@ class Migration(migrations.Migration):
         ),
         migrations.AlterUniqueTogether(
             name='publishedmetadata',
-            unique_together={('publication', 'file'), ('publication', 'relative_path')},
+            unique_together={('publication', 'relative_path'), ('publication', 'file')},
```

Comment:
Interesting... it seems like the order is random and it's confusing because there were no changes to the uniqueness constraints :\

Reply:
Can we just not have this line included in the diff?

Reply:
I'm LGTM if we can remove this 1 line.